modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -515,6 +521,11 @@ bool EditorFileSystemImportFormatSupportQueryBlend::query() {
 		browse_dialog->connect("dir_selected", callable_mp(this, &EditorFileSystemImportFormatSupportQueryBlend::_select_install));
 
 		EditorNode::get_singleton()->get_gui_base()->add_child(browse_dialog);
+
+		// Update icons.
+		// (This is an hack because we can't rely on notifications here as we don't received them.
```

Comment:
```suggestion
		// (This is a hack because we can't rely on notifications here as we don't receive them.
```

Reply:
Dare I say the parentheses are superfluous too (nitpick)